examples/pid-json-example-payload.json

```diff
@@ -30,5 +33,8 @@
   "given_name": "Mario",
   "family_name": "Rossi",
   "birth_date": "1980-01-10",
-  "personal_administrative_number": "TINIT-XXXXXXXXXXXXXXXX"
+  "birth_place": "Roma",
+  "nationality": "IT",
+  "personal_administrative_number": "TINIT-XXXXXXXXXXXXXXXX",
+  "tax_id_code": "TINIT-XXXXXXXXXXXXXXXX"
```

Comment:
```suggestion
```
if tax_id_code and personal_administrative_number brings the same value it would be better stop using tax_id_code.

otherwise if tax_id_code is valued with codice fiscale and personal_administrative_number with id anpr it would make more sense